cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/CruiseControlApi.java

```diff
@@ -20,11 +20,6 @@ public interface CruiseControlApi {
      */
     String CC_REST_API_PROGRESS_KEY = "progress";
 
-    /**
-     * User ID header key
-     */
-    String CC_REST_API_USER_ID_HEADER = "User-Task-ID";
```

Comment:
Are the above string not headers and some other keys instead?

Reply:
Yes. They are not headers, but field names used by Vertx JsonObject to extract content from the CC response. They wouldn't be needed with the new client, as you would get a POJO with getters.

Reply:
Ok, great. Thanks.